deploy/inference-gateway/epp-aware/resources/cluster-role-binding.yaml

```
@@ -0,0 +1,25 @@
+# limitations under the License.
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pod-read-binding
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: pod-read
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)